app/src/main/java/agency/five/codebase/android/movieapp/ui/component/MovieCard.kt

```diff
@@ -0,0 +1,56 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.R
+import agency.five.codebase.android.movieapp.mock.MoviesMock
+import agency.five.codebase.android.movieapp.model.Movie
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+
+@Composable
+fun MovieCard(
+    movie: Movie,
```

Comment:
It is good that you provided state with Movie, but it would be better if you follow the same example as in ActorCard and CrewItem where you had ViewState.

Here, you can create MovieCardViewState.